crates/wasm-qmd-parser/Cargo.toml

```diff
@@ -12,7 +12,7 @@ default = ["console_error_panic_hook"]
 
 [dependencies]
 quarto-markdown-pandoc = { workspace = true }
-wasm-bindgen = "0.2.84"
+wasm-bindgen = "0.2.89"
```

Comment:
Supposedly this is when wasm-bindgen started working right with extern C
https://github.com/rust9x/rust/blob/19b01bd748a7829a5f42f32eb31b21d2e54d30b7/src/doc/rustc/src/platform-support/wasm32-unknown-unknown.md?plain=1#L293